cypress/support/fragments/requests/requestDetail.js

```diff
@@ -123,6 +133,7 @@ export default {
     InteractorsTools.checkKeyValue(requestInfoSection, 'Position in queue', data.position);
     InteractorsTools.checkKeyValue(requestInfoSection, 'Request level', data.level);
     InteractorsTools.checkKeyValue(requestInfoSection, 'Patron comments', data.comments);
+    InteractorsTools.checkKeyValue(requestInfoSection, 'Cancellation reason', data.reason);
```

Comment:
Did you run existing tests with this method?
Seems like it wiil fail, because data.reason will be undefined

Reply:
I did not quite catch your point, data.reason is provided here:

````
RequestDetail.checkRequestInformation({
        type: REQUEST_TYPES.PAGE,
        status: EditRequest.requestStatuses.CLOSED_CANCELLED,
        level: REQUEST_LEVELS.ITEM,
        **reason**: 'INN-Reach',
      });
````

Reply:
This method is used in another test. And there is not provided

Reply:
@manowarqqq yep, you are right, added a check to handle data.reason